scripts/kcc

```diff
@@ -26,14 +26,24 @@ my @srcFiles = ();
 my @tempFiles = ();
 
 my $distDirectory = dirname(rel2abs($0));
+our $profile;
```

Comment:
@chathhorn this is done so that they can be accessed from the actions on the Getopt::Declare.